cms/envs/test.py

```diff
@@ -320,7 +320,3 @@
 ######### custom courses #########
 INSTALLED_APPS += ('openedx.core.djangoapps.ccxcon',)
 FEATURES['CUSTOM_COURSES_EDX'] = True
-
-# API access management. Necessary so that django-simple-history
-# doesn't break when running pre-test migrations.
-INSTALLED_APPS += ('openedx.core.djangoapps.api_admin',)
```

Comment:
Yes -- sorry, I should have linked to the most recent Jenkins run. The errors look something like this, which I got from devstack:

```
ERROR: Failure: KeyError (u'api_admin')

   Traceback (most recent call last):
    /edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/nose/loader.py line 418 in loadTestsFromName
      addr.filename, addr.module)
    /edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/nose/importer.py line 47 in importFromPath
      return self.importFromDir(dir_path, fqname)
    /edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/nose/importer.py line 94 in importFromDir
      mod = load_module(part_fqname, fh, filename, desc)
    openedx/core/djangoapps/api_admin/tests/test_models.py line 6 in <module>
      from openedx.core.djangoapps.api_admin.models import ApiAccessRequest, ApiAccessConfig
    openedx/core/djangoapps/api_admin/models.py line 15 in <module>
      class ApiAccessRequest(TimeStampedModel):
    /edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/db/models/base.py line 308 in __new__
      new_class._prepare()
    /edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/db/models/base.py line 370 in _prepare
      signals.class_prepared.send(sender=cls)
    /edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/django/dispatch/dispatcher.py line 189 in send
      response = receiver(signal=self, sender=sender, **named)
    /edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/simple_history/models.py line 80 in finalize
      history_model = self.create_history_model(sender)
    /edx/app/edxapp/venvs/edxapp/local/lib/python2.7/site-packages/simple_history/models.py line 112 in create_history_model
      app = apps.app_configs[model._meta.app_label]
   KeyError: u'api_admin'
```
For some reason the simply history migrations are getting picked up in CMS, even though `api_admin` isn't an installed app. I don't know exactly how simple history is implemented, but my guess is that the history migrations happen via metaclass magic at class-loading time. That makes me worried that the only way to get this to work is to either remove simple history (bad) or install `api_admin` for running tests, along with its requisite settings (bad, but less so).